CONTRIBUTORS.txt

```diff
@@ -107,7 +102,7 @@ contributors:
 
 * Aru Sahni: Git ignoring, regex-based ignores
 
-* Mike Frysinger: contributor.
+* Mike Frysinger
```

Comment:
We need to update the contributing documentation. That says something about adding your names here.

Reply:
[cde0c32](https://github.com/PyCQA/pylint/pull/5909/commits/cde0c3232a4e3ee82bb6fabee4676e313d64aae4)

Reply:
I'm going to really automate in #5877 so I'll update there.